repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/features/filters/rackspaceauthuser/RackspaceAuthUserIpIdentityTest.groovy

```diff
@@ -64,21 +84,28 @@ class RackspaceAuthUserIpIdentityTest extends ReposeValveTest {
         and: "Repose will send user from Request body"
         ((Handling) sentRequest).request.getHeaders().findAll("x-pp-user").contains(expectedUser + ";q=0.8")
 
+        and: "Repose will send two values for x-pp-domain"
+        ((Handling) sentRequest).request.getHeaders().findAll("x-domain").size() == 1
+
+        and: "Repose will send 'My Group' for x-pp-domain"
```

Comment:
Yeh. @adrianjgeorge had me change that after I had added the tests... and I forgot to update the comments.
 Fixing now.